
<!DOCTYPE html>
<html lang="en">

    <head>

        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <!-- Bootstrap CSS CDN -->
       <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
             integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
             crossorigin="anonymous">
        <!-- Bootstrap CSS local fallback -->
        <script>
            var test = document.createElement("div")
            test.className = "hidden d-none"

            document.head.appendChild(test)
            var cssLoaded = window.getComputedStyle(test).display === "none"
            document.head.removeChild(test)

            if (!cssLoaded) {
                var link = document.createElement("link");

                link.type = "text/css";
                link.rel = "stylesheet";
                link.href = "bootstrap-4.0.0-dist/css/bootstrap.min.css";

                document.head.appendChild(link);
            }
        </script>
        <link rel="stylesheet" type="text/css" href="css/main.css">
        <link rel="stylesheet" type="text/css" href="css/hover-min.css">
    </head>

    <body>
            <button class="tablink" onclick="openPage('About Me', this, 'grey')" id="defaultOpen">About Me</button>
            <button class="tablink" onclick="openPage('Projects', this, 'grey')">Projects</button>
            <button class="tablink" onclick="openPage('Publications', this, 'grey')">Publications</button>
            <button class="tablink" onclick="openPage('Contact', this, 'grey')">Contact</button>

            <div id="About Me" class="tabcontent">
                <img src="images/headshot_yaklich.jpg" width="300" height="300"
                     alt="Yaklich-Emily" , class="ImageBorder"></a>
                <h1><b>Hello, I am Emily Yaklich!</b></h1>
                <hr>
                <p>I am currently a second year graduate student in the Department of Plant Biology at the
                    University of Georgia. I obtained my B.S. in Genetics, Cell Biology, and Development at
                    the University of Minnesota and subsequently spent three years at the National Institutes
                    of Health (NIH) studying symmetry in 3D protein structures using mathematical software to study biological data, also known as bioinformatics.
                    An example of a top view 3D structure of a <a href="https://www.rcsb.org/structure/1XL6">potassium channel</a> is pictured here:
                    <br style=“line-height:N;”>
                    <img src="images/1xl6_quat_repeats_top_view.png" width="250" height="200", class="center">

                    My strong background in genetics and bioinformatics has set me up wonderfully to study how sunflowers
                    respond to different environmental conditions in my graduate program using genomics and transcriptomics.
                    Also, getting to work with beautiful sunflowers means I draw them any chance I get (using the computer, of course):
                    <br style=“line-height:N;”>
                    <img src="images/python_turtle.gif", class="center">





                </p>
                <hr>
                <p> In my free time I enjoy running, biking, and hiking (basically anything outdoors)!
                <br style=“line-height:N;”>
                <br style=“line-height:N;”>
            </div>


            <div id="Projects" class="tabcontent">
                <h1><b>Projects</b></h1>
                I am sharing some examples of projects that I have worked on whether part of an academic course or in my free time.
                <hr>
                 <h3> <b> Machine learning model to predict transmembrane protein identity based on sequence features</b></h3>

                    <p>Machine learning models can be valuable tools for protein prediction, specifically, which regions
                        of a protein are embedded in the cell membrane or not. I created a model that will be able to
                        classify if a protein sequence is transmembrane (TM) or in the cytosol. This model can predict
                        TM membership with ~93% accuracy. The purpose of creating this model was to build a foundation
                        and familiarize myself with machine learning protocols. I can now compare this model to already
                        existing TM prediction algorithms and then further improve this model to possibly classify
                        specific TM helix regions and/or membrane protein interfaces in my own work. This project was
                        completed as part of a machine learning course. </p>
                <a href="https://github.com/emilyyaklich/final-project" class="myButton" class="hvr-skew">GitHub repository</a>
                <hr>

                <h3> <b>Introduction to Python Tutorial Materials</b></h3>
                <p> In collaboration with my colleagues, we created an <em> Intro to Python</em> tutorial.
                This tutorial outlines some basic concepts and Python use cases in a set of Jupyter Notebooks that can
                be worked through as a group or individually. </p>
                <a href="https://github.com/suhwanplee/Introduction-to-Python" class="myButton">GitHub repository</a>
                <hr>

                <h3><b> Wikipedia Page on C19orf44 (a protein of unknown function)</b></h3>
                  <p>
                      Chromosome 19 open reading frame 44 (C19orf44) is a protein of unknown function. As part of
                      an undergraduate computational biology course, I performed a web-based analysis of this
                      protein in order to gather information on the C19orf44 gene, protein, expression
                      locations and levels, homology and evolution, and protein-protein interactions.
                            </p>
                <a href="https://en.wikipedia.org/wiki/C19orf44" class="myButton">Wikipedia page</a>
                <hr>

                <h3><b> Morris Hoeft Group Website</b></h3>
                  <p>
                      Built the website for Morris Hoeft Group LLC.
                            </p>
                <a href="https://morrishoeft.com/" class="myButton">Website</a>
                <hr>

                <h3> <b>This Website! </b></h3>
                <p> I have been building my personal website as a hobby in my free time.</p>
                <a href="https://github.com/emilyyaklich/emilyyaklich.github.io" class="myButton">GitHub repository</a>
                <hr>


            </div>

            <div id="Publications" class="tabcontent">
                <h1><b>Publications</b></h1>
                <hr>
                <h5> Staritzbichler R, <b>Yaklich E</b>, Sarti E, Ristic N, Hildebrand PW, Forrest LR, <i>AlignMe: an update of
                    the web server for alignment of membrane protein sequences.</i> 2022. Nucleic Acids Research.
                    doi: 10.1093/nar/gkac391</h5>
                The AlignMe webserver is specially designed for the accurate amino acid sequence alignment of membrane
                proteins. Membrane proteins are notoriously difficult to align due to strong evolutionary divergence and
                low amino acid complexity due to hydrophoibc residues within the membrane. The AlignMe server has been around for about a decade,
                but updates to this release include multiple submissions, structural visualization, and anchoring of
                residues. The anchoring of residues allows the user to constrain specific amino acids in the sequence
                alignment based on biochemical, biophysical, or genetic data within the sequence alignment. These updates to
                the AlignMe server make it more user friendly and continues to be a useful resource for the membrane
                protein community.
                <br style=“line-height:N;”>
                <a href="https://academic.oup.com/nar/article/50/W1/W29/6591521" class="myButton">Link to article</a>

                <hr>
                <h5>Staritzbichler R, Sarti E, <b>Yaklich E</b>, Aleksandrova A, Stamm M, Khafizov K, Forrest LR. <i>Refining
                    pairwise sequence alignments of membrane proteins by the incorporation of anchors.</i> 2021. PLoS ONE
                    16(4): e0239881. doi: 10.1371/journal.pone.0239881.</h5>
                Membrane proteins are encoded by 20- 30% of genomes and have diverse functions including signaling,  scaffolding, and transporting molecules. Aligning sequences of membrane proteins can be difficult
                due to the complex hydrophobic membrane environment and low sequence identity. AlignMe
                (www.bioinfo.mpg.de/AlignMe/) is a program optimized for aligning membrane proteins allowing the
                input of evolutionary, transmembrane, and secondary structure predictions. Additionally, AlignMe
                allows for the incorporation of biochemical/biophysical data by using a restraint to match positions
                between two sequences known as an anchor. Historically, this biochemical information is added
                manually after the sequence alignment is complete, but the use of anchors allows non-anchored
                residues to be aligned following the AlignMe algorithm. Incorporating biochemical data through
                the use of anchors in AlignMe is an efficient and reproducible way of modifying alignments
                of membrane protein sequences.
                <br style=“line-height:N;”>
                <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0239881" class="myButton">Link to article</a>
                <hr>
            </div>

            <div id="Contact" class="tabcontent">
                <h1><b>Contact</b></h1>
                <hr>
                <h4>Feel free to reach out if you would like to chat or to see my full curriculum vitae.
                    The best way to reach me is via email at emily.yaklich@gmail.com.</h4>
            </div>


    <script src="my_script.js"></script>

     <!-- jQuery CDN -->
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
            integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
            crossorigin="anonymous"></script>
    <!-- jQuery local fallback -->
    <script>window.jQuery || document.write('<script src="jquery-3.3.1.min.js"><\/script>')</script>

    <!-- Bootstrap JS CDN -->
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"
            integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
            crossorigin="anonymous"></script>
    <!-- Bootstrap JS local fallback -->
    <script>if(typeof($.fn.modal) === 'undefined')
    {document.write('<script src="/bootstrap-4.0.0-dist/js/bootstrap/js/bootstrap.min.js"><\/script>')}</script>

    </body>
</html>
